Jenkinsfile_CNP

```diff
@@ -34,6 +34,7 @@ withPipeline("nodejs", product, component) {
         notifyBuildEvent channel: notificationsChannel, color: 'warning', message: 'Failed to update demo branch'
       }
     }
+    sh "yarn cache clean"
```

Comment:
builds failing constantly without this, not sure whether we should keep it in or what

Reply:
We should not keep it unless that official and temporarily workaround for known problem.

I am saying this since having this would impact our and others build times.

Reply:
Is that one module constantly failing or random?

Reply:
no multiple ones

Reply:
:-(